### PR TITLE
Add components to lookups view

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/10107

## Testing
**URL to test:** 
had to change the metadata and add an array relationship, so unfortunately this won't work on the netlify deploy. 

**Steps to test:**

go to `/moped/dev/lookups#moped-components` and see the components with subcomponents. 

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] ~Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable~

### CODE REVIEW

File: moped-database/metadata/tables.yaml [hunk 1]
```diff
@@ -124,6 +124,14 @@
...
     schema: public
     name: moped_components
+  array_relationships:
+    - name: moped_subcomponents
+      using:
+        foreign_key_constraint_on:
+          column: component_id
+          table:
+            schema: public
+            name: moped_subcomponents
```
Comment (johnclary) [inline moped-database/metadata/tables.yaml:134]: yeesh—thanks for adding this!

File: moped-editor/src/views/dev/LookupsView/settings.js [hunk 2]
```diff
@@ -10,12 +10,25 @@ const subPhaseHandler = (subphases) =>
   ));
 
 /**
- * Parses an array of phase-subphases into an array of subphase names
- * @param {Object[]} subphases - array of moped_subphases objects
- * @returns { JSX } An array of <div>s with the subphase name
+ * Retrieves phase name from phase object
+ * @param {Object} phase - phase object
+ * @returns { string } phase name
```
Comment (johnclary) [inline moped-editor/src/views/dev/LookupsView/settings.js:15]: 🙈

**Review by johnclary — APPROVED** [review]
Thanks for squeezing this one in so quickly! Really nice to have these readily available.

🌬️⛵

**Review by amenity — APPROVED** [review]
(Screensharing with @chiaberry) 
Looks good to me!

**Pull request merged**